### PR TITLE
[RM Ch04] Fix exposed versus Internal

### LINKED ISSUE #2718: [RM Ch04] Errors in 4.1.4.2 Internal SLA capabilities
"CPU allocation ratio" is externally specified.

@walterkozlowski Can we discuss this in the next RM meeting?

### PR DESCRIPTION
**Pull request by pgoyal01**
Fixes #2718 #2719

Moved CPU Allocation Ration and Huge Pages to Expose capabilities

### CODE REVIEW

**Comment by pgoyal01** [comment]
> @pgoyal01 I agree with the proposal, but there is an impact on some references which must be renumbered: i.cap.017 and e.cap.017 (Monitoring)

@karinesevilla I was thinking of not renumbering except where it has specifically changed, as there are references all over the place

**Comment by karinesevilla** [comment]
@pgoyal01 there is still an issue. 
I have just noticed that, later on the chapter, in the section about profiles (Profiles Specifications & Capability Mapping), a table is using the references e.cap.018(SMT) and e.cap.019(vswitch optimisation). 
It can be considered as a bug or inconsistency, as the references e.cap.018 to e.cap. 022 and i.cap.019 to i.cap.020 mentioned for profiles should have been defined before.

**Review by walterkozlowski — APPROVED** [review]

**Review by karinesevilla — APPROVED** [review]

File: doc/ref_model/chapters/chapter04.rst [hunk 1]
```diff
@@ -77,6 +77,8 @@ e.cap.013 SR-IOV over PCI-PT              Yes/No Traditional SR-IOV. These Capab
 e.cap.014 GPU/NPU                         Yes/No Hardware coprocessor. These Capabilities generally require hardware-dependent drivers be injected into workloads
 e.cap.015 SmartNIC                        Yes/No Network Acceleration
 e.cap.016 FPGA/other Acceleration HW      Yes/No These Capabilities generally require hardware-dependent drivers be injected into workloads
+e.cap.023 Huge pages                      Yes/No Indicates if the Cloud Infrastructure supports huge pages
```
Comment (ulikleber) [inline doc/ref_model/chapters/chapter04.rst:80]: The table below in "Profiles Specifications & Capability Mapping" (I didn't see a table number) still lists Huge page as i.cap.18.

Comment (ulikleber) [reply doc/ref_model/chapters/chapter04.rst:80]: I was not talking about table 4-7 but about the table without table number in clause "Profiles Specifications & Capability Mapping" further below.

Comment (ulikleber) [reply doc/ref_model/chapters/chapter04.rst:80]: It is in line 352, which I cannot comment, maybe because it is below the PR content.

Comment (ulikleber) [reply doc/ref_model/chapters/chapter04.rst:80]: Thanks, @pgoyal01

**Review by walterkozlowski — APPROVED** [review]

**Pull request merged**